llvm/test/CMakeLists.txt

```diff
@@ -133,6 +133,7 @@ set(LLVM_TEST_DEPENDS
           opt
           sancov
           sanstats
+          spir-to-ir
```

Comment:
Maybe something like `spv-to-ir-wrapper` would reflect the "conditional" translation functionality better? It just feels like the name in itself is a tad misleading ("Why the new tool? Isn't this done by llvm-spirv already?")

Reply:
Done.